llvm/test/MC/RISCV/bigendian-fixups.s

```diff
@@ -0,0 +1,60 @@
+# RUN: llvm-mc -filetype=obj -triple=riscv32be < %s \
+# RUN:     | llvm-objdump --no-print-imm-hex -M no-aliases -d - \
```

Comment:
Test inline relocations with `llvm-objdump -dr -`. It's important to test the encoding for at least a few insns, so you should omit --no-print-imm-hex